### PR TITLE
Add config for send notifications task

### PR DESCRIPTION
**Pull request by aliveni**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-1096

### Change description ###
Add cron expression for the task which sends messages to notifications queue.
The task should run every 10 minutes.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by aliveni** [comment]
Flux PR https://github.com/hmcts/cnp-flux-config/pull/2741

File: charts/reform-scan-blob-router/values.yaml [hunk 1]
```diff
@@ -31,6 +31,7 @@ java:
     CHECK_NEW_ENVELOPES_CRON: "0 0 * * * *"
     TASK_SCAN_DELAY: "4000" # in millis
     SEND_DAILY_REPORT_CRON: "0 0 18 ? * MON-FRI"
+    SEND_NOTIFICATIONS_CRON: "0 0/10 * * * *"
```
Comment (nitinprabhu) [inline charts/reform-scan-blob-router/values.yaml]: Do we want to override for aat and preview ?

Comment (aliveni) [reply charts/reform-scan-blob-router/values.yaml]: Yeah, will update the PR

Comment (aliveni) [reply charts/reform-scan-blob-router/values.yaml]: Updated preview and aat values.yaml files

Comment (doncem) [reply charts/reform-scan-blob-router/values.yaml]: aat shouldn't be updated but since we already applied such practice for this service - let it be so

**Comment by aliveni** [comment]
Closing the PR as the build is failing

**Pull request closed without merging**